### DISCUSSION TITLE
Support for 22H2 (22621)

### DISCUSSION THREAD

**Discussion by jed84** [valinet/ExplorerPatcher#2194] [General]
Does the latest EP support Win11 22H2 (22621)?

I couldn't find that info at https://github.com/valinet/ExplorerPatcher/

I see some people having EP and Win11 22H2 working, but I am not the lucky one. 

Any official info about this or suggestion where can I find the answer?

**Comment by Poopooracoocoo** [comment]
You'll find which OS builds an EP version has been tested on in its description: https://github.com/valinet/ExplorerPatcher/releases

**Comment by jed84** [comment]
I am on win 22621.2361 having EP 22621.2361.58.1 installed
It doesnt work (taskbar v.10/11 and start menu v11 do not load) at causes crashes.

How can I properly create a bug and attach logs?

**Reply by Amrsatrio** [reply]
You can just report it here. It's very weird indeed.
* What is your Windows locale? Is it English (United States) or some other? Check in `msinfo32`.
* Can you upload your `twinui.pcshell.dll` from system32 and upload it here with PDF extension?
* Do you use any other tweak software such as OldNewExplorer and TranslucentTB?

I tried on my system with that exact build but only OldNewExplorer and it works just fine, everything.

**Comment by jed84** [comment]
After many attempts, I managed to fix EP Win10 taskbar.
What still does not work is EP Win11 start menu. Once I click on the Start button, nothing happens.

- I am on Win11 22621.2361 
- I have Win7+Taskbar Tweaker installed and running
- I had TranslucentTB installed, but now removed
- My Windows local settings from msinfo32: Poland
- twinui.pcshell attached [twinui.pcshell.pdf](https://github.com/valinet/ExplorerPatcher/files/12734719/twinui.pcshell.pdf)

**Reply by Amrsatrio** [reply]
Please check if you experience the same symptoms with this another 2361 user who is experiencing a similar issue. https://github.com/valinet/ExplorerPatcher/discussions/2182#discussioncomment-7105126

I am trying my best to make EP supported on Release Preview builds like these.

**Reply by Amrsatrio** [reply]
The other user in the issue I referred to seems to also be using 7+TT which might be causing a compatibility issue (for now).

**Comment by jed84** [comment]
I uninstalled 7+TT, but the issue keeps going.

The console does not show any errors after I click on the start menu button, however I do have an error related to the symbols:
![obraz](https://github.com/valinet/ExplorerPatcher/assets/47121522/87143881-706e-48ae-be6c-2cfe86a549ae)

**Reply by Amrsatrio** [reply]
Do you have these lines with exact values at the beginning of the console? If some lines are missing then 7+TT definitely left behind some files or you're using another tweak software that modifies the Explorer process.

The issue mostly lies on my new method to get Win+X and Win10 Alt+Tab working on builds that do not have symbols like yours.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/d0ee7e1d-6220-43dc-8d65-f7ed23eacca0)

**Comment by jed84** [comment]
I am getting similar logs:

![obraz](https://github.com/valinet/ExplorerPatcher/assets/47121522/d4ca8253-e29d-4aa8-9ba3-9097ba602262)

**Comment by Amrsatrio** [comment]
Please try reinstalling the latest EP **while being connected to the Internet**. It is needed to download files to make the old start menu work.

**Comment by jed84** [comment]
Thank you so much!
Its working perfectly!
